### PR TITLE
kie-issues#376: DMN Runner context menu options are not appropriate regarding the selected cell

### LINKED ISSUE #376: DMN Runner context menu options are not appropriate regarding the selected cell
One general question, is by design columns can not be added/deleted neither for inputs not outputs table?

### Outputs Data cell
Screenshot 2023-06-30 141958

- Should we offer operations `Insert`,  `Delete`, `Duplicate`, `Reset`? Currently none of them works.
- Should we offer operations like `Copy`,  `Cut`, `Paste`, `Reset`? Currently none of them works. We should make only **Copy** working?

### Inputs Data cell
Screenshot 2023-06-30 141932

- Should we offer operations `Insert`,  `Delete`, `Duplicate`, `Reset`? Currently none of them works.
- Should we offer single operation **Copy**?

### Outputs Header cell
Screenshot 2023-06-30 141926

- Should we offer operations `Insert`,  `Delete`, `Duplicate`, `Reset`? Currently none of them works.
- Should we offer operations like `Copy`,  `Cut`, `Paste`, `Reset`? Currently none of them works. We should make only **Copy** working?

### Inputs Header cell
Screenshot 2023-06-30 141851

- Should we offer operations `Insert`,  `Delete`, `Duplicate`, `Reset`? Currently none of them works.
- Should we offer single operation **Copy**?

### PR DESCRIPTION
**Pull request by jomarko**
Closes: kiegroup/kie-issues#376

### CODE REVIEW

File: packages/unitables/src/bee/UnitablesBeeTable.tsx [hunk 1]
```diff
@@ -103,16 +112,25 @@ export function UnitablesBeeTable({
...
-      BeeTableOperation.RowReset,
-      BeeTableOperation.RowDelete,
-    ];
-  }, []);
+  const allowedOperations = useCallback(
+    (conditions: BeeTableContextMenuAllowedOperationsConditions) => {
+      return [
+        BeeTableOperation.SelectionCopy,
+        BeeTableOperation.SelectionCut,
+        BeeTableOperation.SelectionPaste,
```
Comment (ljmotta) [inline packages/unitables/src/bee/UnitablesBeeTable.tsx]: In the https://github.com/kiegroup/kie-issues/issues/376 I've mentioned that we didn't need any command in the Input Header but after using it, the Copy would be appreciated. So the Cut and Paste should be removed as they're not suppose to work.

**Review by ljmotta — COMMENTED** [review]
@jomarko Thanks for your PR! It looks great. I just have one comment that I've made inline.

File: packages/unitables/src/bee/UnitablesBeeTable.tsx [hunk 2]
```diff
@@ -89,6 +90,14 @@ export function UnitablesBeeTable({
 }: UnitablesBeeTable) {
   const beeTableOperationConfig = useMemo<BeeTableOperationConfig>(
     () => [
+      {
+        group: _.upperCase(i18n.terms.selection),
```
Comment (tiagobento) [inline packages/unitables/src/bee/UnitablesBeeTable.tsx]: I don't think we need to use lodash for this. There's a built-in function that does the same.

File: packages/unitables/src/bee/UnitablesBeeTable.tsx (hunk 1 above)
Comment (jomarko) [reply packages/unitables/src/bee/UnitablesBeeTable.tsx]: @ljmotta thank you for spotting this. I keep `cut` and `paste` for input data cells, as there they work. See my latest changes.

File: packages/unitables/src/bee/UnitablesBeeTable.tsx (hunk 2 above)
Comment (jomarko) [reply packages/unitables/src/bee/UnitablesBeeTable.tsx]: thank you @tiagobento , I think we used `lodash` somehow historically, I replaced its usage on all similar places.

**Review by yesamer — APPROVED** [review]

**Review by ljmotta — APPROVED** [review]

**Pull request merged**